src/main/java/org/micks/champmaker/championships/ChampionshipController.java

```diff
@@ -71,4 +76,19 @@ public void editingChampionship(@PathVariable long champId, @RequestBody Champio
     public void shuffleTeams(@PathVariable long champId) {
         championshipService.shuffleTeams(champId);
     }
+
+    @PostMapping(value = "/{champId}/meals", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public void createMeal(@PathVariable long champId, @RequestBody MealDTO mealDTO) {
+        mealService.createMeal(champId, mealDTO);
+    }
+
+    @PutMapping(value = "/{champId}/meals")
+    public void editMeals(@PathVariable long champId, @RequestBody MealDTO mealDTO) {
+        mealService.editMeals(champId, mealDTO);
```

Comment:
Should be `editMeal` instead of `editMeals` - we are editing just one meal here. Applies both to controller method name and service method name.